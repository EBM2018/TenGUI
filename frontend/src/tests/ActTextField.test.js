import React from 'react';
import renderer from 'react-test-renderer';
import ActTextField from '../react/Widgets/Activity/ActTextField';


test('ActTextField behaves as expected', () => {
  const component = renderer
    .create(<ActTextField id={1} text="This is a test" editResponse={() => {}} />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
