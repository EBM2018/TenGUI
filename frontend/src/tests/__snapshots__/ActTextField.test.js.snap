// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActTextField behaves as expected 1`] = `
<div>
  This is a test
  <div>
    <input
      id="input_response"
      onChange={[Function]}
      type="text"
    />
  </div>
</div>
`;
